### PR TITLE
feat: placeholders added to help me choose experiment

### PR DESCRIPTION
**Pull request by roger-in-kiva**

### CODE REVIEW

**Review by mcstover — APPROVED** [review]

File: src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue [hunk 1]
```diff
@@ -58,6 +59,10 @@ export default {
 	computed: {
 		activeLoan() {
 			return this.selected === this.index;
+		},
+		boxShadow() {
+			if (this.activeLoan) return 'box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.05);';
+			return 'none';
```
Comment (dyersituations) [inline src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue:65]: I think you want to return empty string here. `style="none"` I don't think is valid.

Comment (roger-in-kiva) [reply src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue:65]: Vue style binding renders this as `box-shadow: none;`
https://vuejs.org/guide/essentials/class-and-style.html#binding-inline-styles

Comment (roger-in-kiva) [reply src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue:65]: The wrong one is the first `box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.05);`
Thank you @dyersituations

Comment (dyersituations) [reply src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue:65]: Ah yup, you're using it as a prop of an object. And yeah I was wondering about the first of then. Good catch!

Comment (roger-in-kiva) [reply src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue:65]: Fixed, thanks!

**Review by dyersituations — APPROVED** [review]

**Pull request merged**